effects/netlify/default.nix

```
@@ -22,6 +23,6 @@ mkEffect (args // {
   effectScript = ''
     netlify deploy \
       --auth=$(readSecretString netlify .${secretField}) \
-      ${lib.escapeShellArgs deployArgs}
+      ${lib.escapeShellArgs deployArgs} | tee out
```

```suggestion
      ${lib.escapeShellArgs deployArgs} | tee netlify-result.json
    # netlify does not print a newline after the json output, so we add it to keep the log tidy
    echo
```

 - fix missing newline
 - make the filename a bit more self-explanatory for readers

I know you want to write to `$out`, but when that's implemented, I'd recommend `$out/netlify-result.json` anyway, as that gives you the freedom to add other files in the future.

Non-directory store paths are kinda bad for this reason and because they make the hash part of the file name, which is weird in things like `cp ${./foo} .`.